<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contacto</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(131, 255, 180);</string>
  </property>
  <widget class="QLineEdit" name="txtCedula">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNombre">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtApellido">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTelefono">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCorreo">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 12, 162);</string>
   </property>
   <property name="text">
    <string>Cedula</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 12, 162);</string>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 12, 162);</string>
   </property>
   <property name="text">
    <string>Apellido</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 12, 162);</string>
   </property>
   <property name="text">
    <string>Telefono</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 12, 162);</string>
   </property>
   <property name="text">
    <string>Correo</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblContactos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>691</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="rowCount">
    <number>1</number>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="btnAgregar">
   <property name="geometry">
    <rect>
     <x>594</x>
     <y>32</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(15, 158, 51);
color: rgb(251, 255, 12);</string>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnModificar">
   <property name="geometry">
    <rect>
     <x>594</x>
     <y>80</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(15, 158, 51);
color: rgb(251, 255, 12);</string>
   </property>
   <property name="text">
    <string>Modificar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEliminar">
   <property name="geometry">
    <rect>
     <x>594</x>
     <y>130</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(15, 158, 51);
color: rgb(251, 255, 12);</string>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>594</x>
     <y>180</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(15, 158, 51);
color: rgb(251, 255, 12);</string>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
